string model_name   # the name of the model to predict
string instruction  # the instruction sended to model for predicton
string unnorm_key   # the dataset of the scene
float32[] source_action

---

bool predict_ret         # result of prediction
float32[] action_flat     # the flatten predicted action
uint8[] action_shape      # the shape of the predicted action: [num_action, action_dim]